examples/rcbilling-demo/.gitignore

```diff
@@ -29,3 +29,5 @@ src/.env
 /playwright-report/
 /blob-report/
 /playwright/.cache/
+
+yarn.lock
```

Comment:
We want to ignore the yarn.lock file only in the demo app.
The sdk is going to use yarn until we can resolve some dependencies issues with npm.